.aws/src/main.ts

```diff
@@ -95,21 +128,28 @@ class SnowplowSharedConsumerStack extends TerraformStack {
     });
   }
 
+  /**
+   * Create SQS subscription for the SNS.
+   * @param sqsLambda SQS integrated with the snowplow-consumer-lambda
+   * @param snsTopicArn topic the SQS wants to subscribe to.
+   * @param snsTopicDlq the DLQ to which the messages will be forwarded if SQS is down
+   * @param topicName topic we want to subscribe to.
+   * @private
+   */
   private subscribeSqsToSnsTopic(
     sqsLambda: SQSConsumerLambda,
-    snsTopicArn: string
+    snsTopicDlq: sqs.SqsQueue,
+    snsTopicArn: string,
+    topicName: string
   ) {
-    // Subscribe to SNS topic with user-related events
     // This Topic already exists and is managed elsewhere
-    new ApplicationSqsSnsTopicSubscription(this, 'user-event-subscription', {
-      name: 'Shared-Snowplow-Consumer-Events',
-      snsTopicArn,
-      sqsQueue: sqsLambda.construct.sqsQueueResource,
-      tags: { environment: config.environment, service: config.name },
-      dependsOn: [
-        sqsLambda.construct.lambda.versionedLambda,
-        sqsLambda.construct.sqsQueueResource as sqs.SqsQueue,
-      ],
+    return new sns.SnsTopicSubscription(this, `${topicName}-sns-subscription`, {
+      topicArn: snsTopicArn,
+      protocol: 'sqs',
+      endpoint: sqsLambda.construct.applicationSqsQueue.sqsQueue.arn,
+      redrivePolicy: JSON.stringify({
```

Comment:
Add deleted line with tags back for easier monitoring

Reply:
the aws doesn't seem to have an option to add `tag` for class `SnsTopicSubscription`